nuxeo-drive-client/nxdrive/engine/dao/sqlite.py

```diff
@@ -1522,5 +1522,13 @@ def insert_update_user_info(self, userid, firstName, lastName):
             con.commit()
 
     def get_user_info(self, userid):
-        con = self._get_read_connection(factory=CustomRow).cursor()
-        return con.execute("SELECT * FROM Users WHERE user_id=?",(userid,)).fetchone()
+        c = self._get_read_connection(factory=CustomRow).cursor()
+        return c.execute("SELECT * FROM Users WHERE user_id=?",(userid,)).fetchone()
+
+    def get_next_user_to_resolve(self):
```

Comment:
better to retrieve all "stale" users in one scoop, e.g.
def get_all_users_to_resolve(self, refreshed_at):
  """Get all users where 'last_refreshed' < refreshed_at"""
  return c.execute(SELECT * from Users WHERE last_refreshed < ?", (refreshed_at,)).fetchall()